### ISSUE TITLE
Production ready

### ISSUE THREAD

**Issue by nonfungibletunji** [colibri-software/locomotive_plugins#2]
Have you used this gem in a production environment. Will the changes to locomotivecms v3 change this gem structure?

**Comment by greeneca** [comment]
Yes we are currently using it in our production app serving up about 30 sites. Hopefully The new changes will not affect any plugins developed too much but it will probably require this gem to be changed.

**Comment by nonfungibletunji** [comment]
Thanks for the reply. I am going to try the basic authentication. I need to be able to authenticate users/people and have them update a profile and protected pages, password resets for lost passwords, etc.

S. Tunji Turner
Sightuary :: @sightuary

**Comment by greeneca** [comment]
The basic auth plugin won't do that for you unfortunately. It will just let you do http basic auth on some of your pages. We have an Identity Plugin that will do most of that. I will see if I can open source it.

**Comment by nonfungibletunji** [comment]
Thx. I will help contribute to fixes and testing. Any timetable? Or email to take this conversation offline.

S. Tunji Turner
Sightuary :: @sightuary

**Comment by greeneca** [comment]
I have open sourced the plugin. You can find it here https://github.com/colibri-software/identity_plugin

**Comment by nonfungibletunji** [comment]
gem 'locomotive_plugins', '~> 1.0', :git => 'git://github.com/colibri-software/locomotive_plugins.git'

was needed to get the latest version  with the HAML updates, do i need :require => 'locomotive/plugin'

**Comment by greeneca** [comment]
The locomotive plugins gem should be included in the gemspec of any plugins
you generate. It dosen't need to be included in your Gemfile of your main
engine app. I usually add require 'locomotive_plugins' in the main file in
the gem lib folder.

Charlie Greene
greeneca@gmail.com

**Comment by nonfungibletunji** [comment]
so if i was using the identity plugin, that would be included in the gemfile of the main app.

**Comment by greeneca** [comment]
Yes you need to include the identity plugin in the main app gem file under
the group locomotive_plugins

Charlie Greene
greeneca@gmail.com
